src/main/java/com/neuvector/Scanner.java

```diff
@@ -329,6 +332,30 @@ private static String maskCredential(String message, String credential){
         return message.replace(credential, "******");
     }
 
+    private static String getDockerUserAndGroup() {
+        String user = executeBashCommand("id -u");
+        String group = executeBashCommand("id -g");
+        return String.format("%s:%s", user,group);
+    }
+
+    private static String executeBashCommand(String command) {
```

Comment:
The name is misleading. You run a command in a separate process (OS dependent). There's no guarantee it's running in `bash`.

Reply:
Done, I closed the buffer in order to avoid nested try ba8aeab419040c259c73099b6824fc7e90260291